CMakeLists.txt

```diff
@@ -107,7 +107,7 @@ if (NOT BUILD_WITH_LIB STREQUAL "CUDA")
 
   if(GPU_TARGETS STREQUAL "all")
     rocm_check_target_ids(DEFAULT_AMDGPU_TARGETS
-      TARGETS "gfx803;gfx900:xnack-;gfx906:xnack-;gfx908:xnack-;gfx90a:xnack-;gfx90a:xnack+;gfx940;gfx941;gfx942;gfx1030;gfx1100;gfx1101;gfx1102;gfx1200;gfx1201"
+      TARGETS "gfx1101;gfx1151"
```

Comment:
It looks like the other targets accidentally got removed here.

Reply:
Wow, missed that one.  Thanks for catching that.  Fixed